src/metadataGeneration/typeResolver.ts

```diff
@@ -657,7 +657,7 @@ export class TypeResolver {
 
         const moduleDeclaration = node as ts.ModuleDeclaration | ts.EnumDeclaration;
         return (moduleDeclaration.name as ts.Identifier).text.toLowerCase() === leftmostName.toLowerCase();
-      }) as Array<ts.ModuleDeclaration | ts.EnumDeclaration>;
+      }) as [ts.ModuleDeclaration | ts.EnumDeclaration];
```

Comment:
are those types equivalent ? 

wouldn't it be : 

```suggestion
      }) as (ts.ModuleDeclaration | ts.EnumDeclaration)[];
```

?

Reply:
Yes! Good catch. Updated!